aikau/src/main/resources/alfresco/documentlibrary/views/AlfFilmStripView.js

```diff
@@ -165,10 +189,18 @@ define(["dojo/_base/declare",
          // NOTE: Any previous previews should have been destroyed, but empty the previewNode just to be on the safe side
          //       TODO: Possible memory leak to investigate here, because the call to empy the node *is* required.
          domConstruct.empty(this.previewNode);
+
+         // Process the widgets for content in order to swap in instance tokens such as the
+         // heightMode and heightAdjustment...
+         var clonedWidgetsForContent = lang.clone(this.widgetsForContent);
+         // this.processObject(["processInstanceTokens"], clonedWidgetsForContent);
+
          this.contentCarousel = new DocumentCarousel({
```

Comment:
Not possible in this case because of the overridden createWidget function in the mixed in _MultiItemRendererMixin mixin.